plone-5.2.x.cfg

```diff
@@ -9,6 +9,7 @@ versions=versions
 plone.restapi =
 plone.namedfile = 5.3.1
 astunparse = 1.6.2
+pep517 = 0.9.1
```

Comment:
This was duplicated after the master merge. See:

https://github.com/plone/plone.restapi/blob/64efe43ea48695902e8579fc5ec9d1d375f03238/plone-5.2.x.cfg#L26

Reply:
oh ok, thanks for the heads up